resolwe/flow/tests/test_executors.py

```diff
@@ -297,8 +294,8 @@ def start_processing(data):
         self.assertEqual(data.worker.status, Worker.STATUS_PROCESSING)
         data.worker.terminate()
 
-        # Give it max 5 seconds to terminate.
-        for _ in range(50):
+        # Give it max 10 seconds to terminate.
```

Comment:
Improve the commit message to make it obvious that it reffers to tests.

Reply:
Done.